llvm/test/lit.cfg.py

```diff
@@ -64,6 +64,8 @@
     config.excludes.append("LoopVectorize")
     # exclude UpdateTestChecks - they fail because of inserted prof annotations
     config.excludes.append("UpdateTestChecks")
+    # issue #166655
```

Comment:
Nit, TODO syntax given it seems like maybe we make them compliant eventually? Eg `TODO(#166555): Reenable`?